pkg/lib/patternfly/patternfly-5-overrides.scss

```diff
@@ -531,7 +531,7 @@ select.pf-v5-c-form-control {
 // intended mainly for PF PageSection elements (pf-v5-c-page__main-section).
 // It's similar to adding padding={{ default: 'noPadding', sm: 'padding' }},
 // except this only affects the sides, not the top and bottom.
-@media screen and (max-width: $pf-v5-global--breakpoint--sm) {
+@media screen and (max-width: $pf-global--breakpoint--sm) {
```

Comment:
It fails to build everywhere. Why would `pf-v5-global--breakpoint--sm` not exist? It may not exist in *podman* as that uses an older PF5 (see https://github.com/cockpit-project/cockpit-podman/pull/1299), but it is certainly correct for cockpit.

Reply:
No, we never had it! But yeah found the issue locally and fixing.